packages/story-editor/src/app/story/effects/useLoadStory.js

```diff
@@ -100,6 +103,10 @@ function useLoadStory({ storyId, shouldLoad, restore, isDemo }) {
           url: embedded?.['wp:publisherlogo']?.[0]?.source_url || '',
         };
 
+        if ('wp:term' in embedded) {
```

Comment:
There will need read values in the link to check if the current use assign / create terms. See
<img width="245" alt="Screenshot 2021-09-13 at 15 05 03" src="https://user-images.githubusercontent.com/237508/133098301-f4bfc984-3d6d-46fb-bcf7-ac2778a27f9a.png">

If user does not have assign / create action link here, the assign / create will fail. So these options will need to hidden.

Reply:
More context https://github.com/google/web-stories-wp/pull/8591 

and https://github.com/google/web-stories-wp/blob/f62effbfc605a3e2a4ddbc8cfd9fa707db28c57f/packages/story-editor/src/app/story/effects/useLoadStory.js#L65-L74

So we will need entries in this object like this `hasAssignTagAction` and `hasCreateTagAction`. But they will need to dynamic of course.

Reply:
know we already talked on slack, but posting new ticket for this here for book keeping. Thanks again for flagging this 🙌 

https://app.zenhub.com/workspaces/web-stories-5e94d3aced449034e1e9b226/issues/google/web-stories-wp/9011